spark-2.0-on-suse/azuredeploy.json

```diff
@@ -23,6 +23,57 @@
 	  "minValue": 1,
       "maxValue": 50
     },
+    "sparkSlavesSize": {
```

Comment:
Hi, I have seen many different templates implements a "dropdown" when executing template, is it mandatory to use a variable?